stl/inc/format

```diff
@@ -716,8 +827,7 @@ public:
         if (_Sv.size() > 4) {
             throw format_error("Invalid fill (too long).");
         }
-        _STD fill(_Specs._Fill, _Specs._Fill + 4, _CharT{});
-        _STD copy(_Sv.begin(), _Sv.end(), _Specs._Fill);
+        _STD fill(_STD copy(_Sv.begin(), _Sv.end(), _Specs._Fill), _STD end(_Specs._Fill), _CharT{});

```

Comment:
In `<chrono>`, @mnatsuhara just added `_Chrono_format_specs::_Fill` and `_Chrono_specs_setter::_On_fill` which should be changed accordingly. (Does all of that newly added character-iterating logic need to be changed accordingly?)